src/python/pants/binaries/executable_pex_tool.py

```diff
@@ -9,10 +9,14 @@
 from pex.pex_info import PexInfo
 
 from pants.backend.python.subsystems.pex_build_util import PexBuilderWrapper
+from pants.base.deprecated import deprecated_module
 from pants.subsystem.subsystem import Subsystem
 from pants.util.dirutil import is_executable, safe_concurrent_creation
 
 
+deprecated_module('1.19.0.dev0', 'Use PythonToolBase and PythonToolPrepBase instead!')
```

Comment:
It doesn't look like this was ever public API. I think we can just nuke it outright, no?

Reply:
It is true that we never marked it `:API: public`, which I had not considered. I have zero qualms killing it outright.